src/main.rs

```diff
@@ -78,8 +78,12 @@ struct Args {
     render: Option<String>,
 
     /// Override the operating system [linux, osx, sunos, windows]
+    #[clap(short = 'p', long = "platform", requires = "command")]
```

Comment:
Maybe add a `short_alias` and `alias` to `-o`/`--os` instead of having two separate fields

Reply:
On second thought, that would stand in the way of the deprecation message...